src/CSI.IBTA.UserService/Services/EmployersService.cs

```diff
@@ -256,7 +257,29 @@ public async Task<GenericResponse<SettingsDto[]>> UpdateEmployerSettings(int emp
             await _unitOfWork.CompleteAsync();
 
             return new GenericResponse<SettingsDto[]>(null, e.Settings.Select(s => new SettingsDto(s.Condition, s.State)).ToArray());
+        }
+
+        public async Task<GenericResponse<SettingsWithEmployerStateDto>> UpdateEmployerClaimSetting(int employerId, UpdateClaimSettingDto updateClaimSettingDto)
+        {
+            var employer = await _unitOfWork.Employers.Include(e => e.Settings)
+                .FirstOrDefaultAsync(s => s.Id == employerId);
```

Comment:
FirstOrDefault assumes that could be many Employers with the same employerId, which is not true
SingleOrDefault is more correct.